src/libPMacc/pmaccConfig.cmake

```diff
@@ -0,0 +1,146 @@
+# - Config file for the pmacc package
```

Comment:
Please add the PMacc license header

This config looks like a many parts of the picongpu cmake file are copied in. If so please add the original authors and your name